source/admin/maven-mirror.rst

```diff
@@ -0,0 +1,82 @@
+Maven mirroring
+===============
+
+Mirroring a maven repository requires downloading all of the artifacts available in the source repository
+(not only .pom, .jar files but also jars with dependencies, jars containing sources and javadocs) and then
+pushing them all using maven to the target repository.
+
+The https://github.com/tungsten-infra/ci-utils repository contains all script and configuration files needed
+to go through the procedure. The steps described below are written for mirroring sdnvcenter.englab.juniper.net/vco-repo
```

Comment:
This (sdnvcenter...) is an uninmportant url, shouldn't be mentioned in the docs.